src/room/GroupCallView.tsx

```diff
@@ -222,9 +222,10 @@ export function GroupCallView({
     return <ErrorView error={error} />;
   } else if (state === GroupCallState.Entered && userChoices) {
     return (
-      <ActiveCall
-        groupCall={groupCall}
+      <OpenIDLoader
```

Comment:
This makes it hard to read because it is not immediately obvious that the active call is inside this wrapper.

Reply:
Yeah, possibly I could add the call as a child & have it set a param or put it in the context

Reply:
OK yeah, this is cleaner.

Reply:
thanks.